### PR TITLE
feat: uppercase theme option

### LINKED ISSUE #289: add uppercase titles theme option

### PR DESCRIPTION
**Pull request by markjjackson**
resolves #289

### CODE REVIEW

File: packages/react/src/components/Campaign/Form.js [hunk 1]
```diff
@@ -161,6 +166,8 @@ const Button = styled(_Button)`
   color: #fff;
   margin-top: 40px;
   width: auto;
+  text-transform: ${({ theme }) =>
+    theme.uppercaseTitles ? 'uppercase' : 'none'};
```
Comment (maartenbusstra) [inline packages/react/src/components/Campaign/Form.js]: I wouldn't use crane's theme in the react package. Let crane handle this internally, and react just consuming crane comps.

File: packages/react/src/components/Campaign/Proposition.js [hunk 2]
```diff
@@ -40,6 +44,8 @@ const ToggleButton = styled.div`
   line-height: 28px;
   padding: 20px 0;
   font-family: ${({ theme }) => theme.headerFontFamily};
+  text-transform: ${({ theme }) =>
+    theme.uppercaseTitles ? 'uppercase' : 'none'};
```
Comment (maartenbusstra) [inline packages/react/src/components/Campaign/Proposition.js]: Same here.

File: packages/react/src/components/Campaign/Form.js (hunk 1 above)
Comment (markjjackson) [reply packages/react/src/components/Campaign/Form.js]: We're already using the theme lots in that package. Maybe factoring it out is something for later.

File: packages/react/src/components/Campaign/Proposition.js (hunk 2 above)
Comment (markjjackson) [reply packages/react/src/components/Campaign/Proposition.js]: This isn't a crane element.

Comment (markjjackson) [reply packages/react/src/components/Campaign/Proposition.js]: Plus, we already use the theme throughout the React package. If that is something to refactor out I think it is outside of the scope of this PR.

**Pull request merged**